.stylelintignore

```diff
@@ -5,3 +5,4 @@ src/renderer/videoJS.css
 dist/
 static/
 node_modules/
+src/renderer/components/ft-video-player
```

Comment:
Is there a reason that this is ignored?

Reply:
The intent was to avoid mucking up the video player with flipping the directionality of elements, which seems to be how YouTube handles it as well (except for things like the caption menu items, which we have as centered in FT). But you bring a good point, in that this linting should just be disabled for this specific rule: implemented now.

On an unrelated note, I did keep VideoJS.css in the ignore file for that reason as well & because it was there previously, but we could just disable the logical property linting rule there. Is there a reason why we wanted this file to be ignored by Stylelint? It should not be super relevant with a replacement on the horizon, but I couldn't find a rationale anywhere on the original PR.

Reply:
Iirc I ignored VideoJS.css because a lot of that file is from videojs (with some customizations added). So, I was thinking it might be desirable to move our custom css to a different file and use the css directly from videojs

Reply:
I didn't know that, thanks for the info. That's reasonable. Do you have any other questions or concerns with the PR?